### ISSUE TITLE
Validation Check in db and Change user-information function

### ISSUE THREAD

**Issue by JacobYim** [JacobYim/CSE442-BuyNSell#109]
Validation Check in db is need for Change user-information function.

When the user change their information, the backup information is need because the connection could be failed due to internet connection error or etc.

therefore, make the validation check variable which is boolean variable in database table to maintain the backup information.

**Comment by JacobYim** [comment]
db schema has been updated at `account-setting-db` branch

```
create table user_profile(
    user_id serial primary key,
    fname varchar(24) not null,
    lname varchar(12) not null,
    ubid varchar(12) unique not null,
    email varchar(50) unique not null,
    password varchar(60) not null,
    address1 varchar(60) not null,
    address2 varchar(60),
    city varchar(60),
    zip int not null,
    states varchar(2) not null,
    file_path varchar(60),
    available BOOLEAN not null
);
```

**Comment by JacobYim** [comment]
This is resolved by #110
